lib/osf-components/addon/components/registries/registration-list/card/template.hbs

```diff
@@ -32,6 +32,17 @@
                     </OsfLink>
                 {{/if}}
             </h3>
+            {{#if @registration.embargoEndDate}}
+                <div local-class='embargoStatus'>
+                    <span>{{t 'registries.registrationList.embargoEndDate'}}</span>
+                    <span> 
+                        {{t 'registries.registrationList.EmbargoEndPrefix'}}
+                        {{format-date @registration.embargoEndDate}}
+                    </span>
+                </div>
+            {{else}}
```

Comment:
This may be a question for Product team, but I think this `{{else}}` is a little too broad of a condition. I think something like `{{else if (not @registration.withdrawn)}}` may be a bit more appropriate, but not sure.

Reply:
Would it show up on that dashboard view if it's withdrawn?

Reply:
Yea, under the "Submitted" tab, we show "Public", "Embargo", "Rejected" and "Withdrawn"

Reply:
So maybe the `else if` condition should check if it's not withdrawn and the reviewsState is not `rejected`?

Reply:
@futa-ikeda I've updated the PR with the changes you requested